debian/cart-docs.docs

```diff
@@ -0,0 +1,2 @@
+README.Debian
+README.source
```

Comment:
This should also include README.env from cart/

Reply:
README.env is not installed by the "install" build step, so the Debian install file will not find it.
The README.Debian and README.source were debian template files that were removed earlier in the review process, so need to be removed from this file.

So first the daos install procedure has to put the README.env somewhere and then this file needs to be updated to look in that relative location.